doc/src/tutorial/manipulation.rst

```diff
@@ -483,3 +483,49 @@ traversals easy.  We could have also written our algorithm as
   have ``-x`` or ``1/x``.  It is also done for speed efficiency because
   singletonized objects can be compared by ``is``.  The unique objects for
   each singletonized class can be accessed from the ``S`` object.
+
+``UnevaluatedExpr``
+===================
```

Comment:
The tutorial should not be about the class `UnevaluatedExpr`, rather about generic methods to keep expressions unevaluated. I think we should also mention `evaluate=False`.